source/bao_hyp/config.rst

```diff
@@ -529,6 +529,46 @@ However, this mechanism may not take effect if the physical mapping feature is e
 specific memory region. Cache coloring exclusively operates in virtual memory systems, i.e.,
 systems featuring Memory Management Units (MMUs) for address translation.
 
+Shared Memory Configuration
+---------------------------
+.. _Shared Memory Configuration:
+
+In Bao's configuration multiple (``shmem0``, ``shmem1``, ...) or no shared memory can be set. If
+at least one shared memory is defined, an  ID is assigned to which the shared memory is
+associated. Later, this ID can be associated with an IPC in the multi-guest configuration.
```

Comment:
```suggestion
In Bao's configuration multiple (``shmem0``, ``shmem1``, ...) or no shared memory can be set. An  ID is assigned to each shared memory object. Later, this ID can be linked to an IPC in the multi-guest configuration.
```

Reply:
Thanks! Fixed  in commit https://github.com/bao-project/bao-docs/pull/78/commits/33652e5f3d7a0867ce9cda4be518f5496cf1078b.